### ISSUE TITLE
Add a {DelayToMousclick} as sequence tag

### ISSUE THREAD

**Issue by Mannshoch** [keepassxreboot/keepassxc#6548]
## Summary
Please add MouseClicck as delay event to the [Autotype sequences ](https://keepassxc.org/docs/KeePassXC_UserGuide.html#_configure_auto_type_sequences) 

## Examples

- [ ] Add a {DelayToClick left} or {Delay MousLeft} as a possible sequence to KeepassXC
  - [ ] Attach a Keepass Icon to the mouse until a click is done and the sequence ended or until a countdown for clicking a field ends.
  - [ ] add middle, right, scrolUp, scrolDown also

## Context
I have to use a bad designed WebApp which request for a Login. It seems It is not able focus the username or password Field by keyboard. I need a solution for that

**Comment by michaelk83** [comment]
There's a global [keyboard shortcut](https://github.com/keepassxreboot/keepassxc/blob/develop/docs/topics/KeyboardShortcuts.adoc) to trigger auto-type, and I think the browser extension has a way to trigger it from the browser.

Also, it may be helpful to know which web app is causing the difficulty.

**Comment by Mannshoch** [comment]
The webapp is inside a proprietary application. It lose the Textbox focus in the moment I click somwhere, even two ALT+TAB do lose the focus. I normaly use the global keyboard shortcut . But here it do not work. As far I see in this case the focus switch back to the main application.

**Comment by JohnLGalt** [comment]
Is it a private app or can others test and see if there is a workaround?

**Comment by Mannshoch** [comment]
It's the cloud storage app (like onedrive or nextcloud) from https://www.3ds.com/3dexperience
Not sure if there is a Free use possibility. 
If you wish I could right-click inside the app and show the source from the html and post it here.

**Comment by michaelk83** [comment]
As a workaround, you can try `{DELAY X}`. Set `X` high enough so you have time to focus the field.
> {DELAY X} | Delay typing start by X milliseconds

Another option is to use regular copy-paste.

**Comment by Mannshoch** [comment]
At moment I use the delay 4000 It works most of the time.

**Comment by DMaxter** [comment]
Is there any update on this issue?

**Comment by droidmonkey** [comment]
Not likely to be implemented to be honest

**Comment by DMaxter** [comment]
Came here from the Citrix-RDP issue

Isn't there a way to avoid the issue by having a setting which allows the user to choose if it wants to hide the window when auto-typing?

**Comment by droidmonkey** [comment]
Not sure what you are talking about, tbh.

**Comment by DMaxter** [comment]
When we use autotype, the KeepassXC windows is minimized, and from what I understood that was the issue in #3661

As an alternative to what is being proposed here (add the new sequence tag), could there be an option in the KeepassXC settings where we could disable this behaviour (disable the minimization of the window when autotyping)?

**Comment by droidmonkey** [comment]
No, we can not disable minimization, and that is not the problem here. The referenced issue is very old, and that discussion was ill-informed. The problem exists with the citric client in that when focus is passed to the client, which we do explicitly now as of 2.7.0, it doesn't actually capture the keyboard focus to the active RDP session. There really isn't much we can do about this. Recommend just adding a delay to the beginning of your Auto-Type sequences to allow for a mouse click.

**Comment by DMaxter** [comment]
But if we use the autotype with Keypass (not KeypassX or XC) the input is passed to the citrix client and it works just fine, and that's very strange
